pkg/controller/dataimportcron-controller.go

```diff
@@ -813,6 +813,10 @@ func (r *DataImportCronReconciler) newCronJob(cron *cdiv1.DataImportCron) (*batc
 		volumes = append(volumes, createConfigMapVolume(ProxyCertVolName, volName))
 	}
 
+	if len(volumes) == 0 {
```

Comment:
instead of `volumes := []corev1.Volume{}` above you can do `var volumes []corev1.Volume` which will make it nil. Append works fine with a nil pointer. Then you don't have to do this.

Reply:
right:) fixed.